app/controllers/users/piv_cac_authentication_setup_controller.rb

```diff
@@ -14,13 +14,13 @@ class PivCacAuthenticationSetupController < ApplicationController
     before_action :cap_piv_cac_count, only: %i[new submit_new_piv_cac]
 
     def new
-      if params.key?(:token)
-        process_piv_cac_setup
-      elsif flash[:error_type].present?
-        render_error
-      else
-        render_prompt
-      end
+      @add_prompt = t('headings.piv_cac_login.add')
+      handle_new
+    end
+
+    def new_required
+      @add_prompt = t('headings.piv_cac_login.add_required')
```

Comment:
I see what you mean about separate routes, but I think that we could do just one.

Since the data we're keying off for this change lives in the session (via the `service_provider_request`) I think that we could just make this branch inside the one route based on that.

ex


```ruby
@add_prompt = if service_provider_request.requires_pivcac?
  t('headings.piv_cac_login.add_required')
else
  t('headings.piv_cac_login.add')
end
```

I wouldn't put the method on the service provider request like that (I'd put it on `Aal3Policy` maybe but same idea

Reply:
The proposed ternary style fails the current Rubocop config:

```
app/controllers/users/piv_cac_authentication_setup_controller.rb:18:9: C: Layout/IndentationWidth: Use 2 (not -12) spaces for indentation.
        t('headings.piv_cac_login.add_required')
        ^^^^^^^^^^^^
app/controllers/users/piv_cac_authentication_setup_controller.rb:19:7: C: Layout/ElseAlignment: Align else with if.
      else
      ^^^^
app/controllers/users/piv_cac_authentication_setup_controller.rb:21:7: W: Layout/EndAlignment: end at 21, 6 is not aligned with if at 17, 20.
      end
      ^^^
```
but it does allow this (which I find easy to read):

```
      @add_prompt =
        if force_piv_cac_setup?
          t('headings.piv_cac_login.add_required')
        else
          t('headings.piv_cac_login.add')
        end
```